apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.ts

```diff
@@ -73,13 +74,25 @@ router.get(
       ).toString();
     }
 
+    let assessmentGHLink: string | null = null;
+    if (res.locals.course.example_course) {
+      // The example course is not found at the root of its repository, so its path is hardcoded
+      assessmentGHLink = `https://github.com/PrairieLearn/PrairieLearn/tree/master/exampleCourse/courseInstances/SectionA/assessments/${res.locals.assessment.tid}`;
+    } else if (res.locals.course.repository) {
+      const githubPrefix = httpPrefixForCourseRepo(res.locals.course.repository);
+      if (githubPrefix) {
+        assessmentGHLink = `${githubPrefix}/tree/${res.locals.course.branch}/courseInstances/${res.locals.course_instance.long_name}/assessments/${res.locals.assessment.tid}`;
+      }
+    }
```

Comment:
I have a [WIP branch](https://github.com/PrairieLearn/PrairieLearn/tree/better-github-links) that'll let us reuse this across branches, I can update it to build on your changes from this PR!